src/app/register/components/form-visibility/form-visibility.component.html

```diff
@@ -73,4 +79,14 @@ <h3 class="mat-subheading-2" i18n="@@register.visibility">
       >Please choose a default visibility setting.</mat-error
     >
   </p>
+  <p>
```

Comment:
p tag it is not necessary here

Reply:
Nice catch!!!

Reply:
is require for the margins 
![image](https://user-images.githubusercontent.com/2119626/85788594-7503b080-b6ea-11ea-9863-a594cc391f6e.png)